.travis.yml

```diff
@@ -5,4 +5,4 @@
   - sudo rm -f /etc/boto.cfg
   - pip install -r requirements.txt
  script:
-    - python3 model.py
+   - python3 text_classifier.py Test dataset.csv d2v.model joblib_model.pkl
```

Comment:
This is not implemented yet, so what about omitting the Travis part from this PR, and create it in a separate new one when we merge this?

Reply:
Actually, this is implemented but removed with the codes I removed as they related to issue #18 so yes will omit this part as well